tasks/main.yml

```diff
@@ -88,12 +88,14 @@
       tags:
         - "atom-basic"
 
+    - include: "init.yml"
```

Comment:
Hi Jesús, I'd add here the following:

```
    - include: "init.yml"
      when:
        - "atom_auto_init is defined and atom_auto_init|bool"
      tags:
        - "atom-site"
```    
Because we usually deploy sites with the `atom-site` tag. The additional conditional will skip all the init.yml tasks when `atom_auto_init` is not enabled.

Reply:
Thanks!